src/js/components/modal.js

```diff
@@ -136,6 +148,18 @@ class Modal {
     this.bindTrigger()
     this.fillModal()
   }
+
+  getCloseTemplate () {
+    if (this.isStaticModal()) {
+      return $('')
```

Comment:
do we need to return a empty jquery object @claudiosikeda ? just return don't work?